UKHomeOffice/sgmr-service
# SPCR-217 Add responsive tables for mobile

## Description
Update the tables across the service to be responsive without compromising on accessibility, by using the guide from [GitHub - Fenwick17/responsive-accessible-table](https://github.com/Fenwick17/responsive-accessible-table)

## To Test
- Go to a page with a table (Voyage Plans, Pleasure Crafts, People, Check your answers)
- Switch to a mobile screen
- The table layout should change to a more mobile-friendly one

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
